env0/resource_environment.go

```diff
@@ -78,13 +199,23 @@ func resourceEnvironmentRead(ctx context.Context, d *schema.ResourceData, meta i
 func resourceEnvironmentUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
This is a gigantic function. Mind extracting its logic to some other functions? 🙏

Reply:
c5609c4 sure thing